### ISSUE TITLE
C.O.P.S.:  Neverwinter

### ISSUE THREAD

**Issue by SeanMJOBrien** [b5635/the-frozen-north#369]
I know we’re not really police, but we might be tasked to respond to these things, especially if it’s too much for the already overwhelmed Neverwinter Guard forces to handle. Think of all the crazy things a city, with magic, and monsters, and undead. And a plague, might need to respond to... We have a small event of undead breaking out of the graveyard and wandering the streets, How about oozes getting out of the sewer. Or finding that kobolds are using a smuggling tunnel, and nastier things have bullied the kobolds out (and the kobolds are spilling into the streets just trying to get away from everyone) to take over and cause trouble in the city? Search out the kobolds, spread out in groups hiding wherever they can. Learn who pushed them out, evaluate the threat, and neutralize it. Plots? We got plots against the city every day. You hear people shouting while patrolling the docks, a boat has succeeded in breaking free of their enforced mooring in the dead of night, you must board another vessel to head off in pursuit and to act as marines to take and board the ship. Or maybe they have the Wailing Death and you are to destroy the plague ship. Dead of night? A light spell on a part of the ship would help us not lose them in the night. Or lost kittens that went in that hole by the sewer drain? Kitty is fine, but to bring it back you need to reason with, trick, or fight a half-orc brute that wants to keep it, notorious for their skill with a club. Or wading in mud to help a child, forming a string of PCs to pull them free, only to have everyone and the child fall into a sinkhole that’s given way. Protect the child and make it out alive. Break up ANOTHER brawl at the tavern? Many of these should be variable level, or allow for multiple levels of PCs to participate. A notorious pirate is said to be trapped with us all, and they are holed up with their crew in a warehouse. (We need a bugger boat, I mean docks area) Will we get a subdual feature?(edited)

A riot has started in Beggar’s Nest. A crazed cult leader has raised a mob, and they are trying to break free of the city. You have broken up the mob, but the leader and a dozen or more of their cult (mostly brutes and enforcers) have barricades themselves in a burned out house and alleyway. You must  it only fight them, but break up the barricades while you do it so that a larger force of guards and other recruits can fight them more easily. Or it could grow over multiple Events, they rille up enough people to mob the forces guarding the gate. Guards bring in more bodies, and so does the mob. The cult leader gets the mob to bully and loot every part of the city under guise of fighting free of tyranny and freedom from the city and the Wailing Death. But really it’s both crazy, and subtraction is exploiting them to gather wealth in the chaos. Smaller combats and diplomacy, etc check could affect which way some of these sub-Events go.(edited)

I’d like to understand the Event system you’re using so I know it’s capabilities and how you currently use it. Any docs? Comments/instructions in the scripts?

**Comment by b5635** [comment]
All these ideas are really awesome. Some of it is out of scope of the event system but that's okay, we can think of something to implement it. I really dig the idea of some ships escaping quarantine and you having to chase it down! Also the tavern brawl mod where you need to break up fights. Reminds me of a mod in Skyrim that has random people brawling. Any of those ideas you have are great, really. And it'd be a nice way to spice things up in the module.

Here's the link to the events system: [https://github.com/b5635/the-frozen-north/wiki/Adding-a-Random-Event](https://github.com/b5635/the-frozen-north/wiki/Adding-a-Random-Event)

I would delve into the linked scripts in that wiki page and take a look at them. Of those, the wilderness and ranger one are more complex. 

Basically, you want to use the CreateEventCreature to create creatures so that they can be properly cleaned up when an area is refreshed. If we need placeables or something else, we can add it np.
